mono/metadata/assembly.c

```diff
@@ -89,7 +89,7 @@ static char **assemblies_path = NULL;
 /* Contains the list of directories that point to auxiliary GACs */
 static char **extra_gac_paths = NULL;
 
-#ifndef DISABLE_ASSEMBLY_REMAPPING
+#ifndef DISABLE_DESKTOP_LOADER
```

Comment:
I think keeping it consistent with the rest of the enable-minimal is better. There's enough of it around the runtime that we're used to it.